cmd/edenRol.go

```diff
@@ -34,7 +34,7 @@ func newRolRentCmd(cfg *openevec.EdenSetupArgs) *cobra.Command {
 		{
 			Message: "Basic Commands",
 			Commands: []*cobra.Command{
-				newCreateRentCmd(&rolProjectID, cfg),
+				newCreateRentCmd(&rolProjectID),
```

Comment:
Seems `func newRolRentCmd(cfg *openevec.EdenSetupArgs) *cobra.Command {` also may not contain cfg

Reply:
Good catch, fixed